LuaUI/Widgets/gui_s44_enableOldUI.lua

```diff
@@ -0,0 +1,79 @@
+function widget:GetInfo()
+	return {
+		name = "1944 Enable/Disable Old UI",
+		desc = "Smooth enable of legacy UI by turning ON, revert by disabling",
+		author = "PepeAmpere",
+		date = "2018-03-06",
+		license = "MIT",
+		layer = 0,
+		enabled = false, -- not loaded by default
+	}
+end
+
+local MAP_HEIGHT_RATIO = 0.12
+
+-- new UI
+local newUIWidgets = {
+	"1944 notAchili Cursor Tip 3",
+	"1944 notAchili Vote Display",
+	"1944 notAchili ss44 UI",
+	"1944 notAchili Epic menu",
+	"1944 notAchili Minimap",
+	"1944 notAchili Widget Selector",
+	"notAchili Framework",
+	"1944 Tooltip Replacement", -- in both lists, reset
+}
+
+local oldUIWidgets = {
+	"1944 Tooltip Replacement" -- in both lists, reset
+}
+
+function DisableNewUI()
+	for i = 1, #newUIWidgets do
+		local wdigetName = newUIWidgets[i]
+		
+		Spring.SendCommands{
```

Comment:
Cannot use `widgetHandler:DisableWidget(wdigetName)`??

BTW, replace `wdigetName` by  `widgetName`

Reply:
* typo is fixed
* `widgetHandler:DisableWidget()` was first thing I tried and it throws me this 

```
[f=0001490] [widgets.lua] Error: Error in Shutdown(): [string "LuaUI/Widgets/gui_s44_enableoldui.lua"]:61: attempt to call method 'DisableWidget' (a nil value)
```

I suspect its because I call it already in shutdown event handler or something similar (so I would need to localize the methods because I proved they present there before the Shutdown). I was experimenting with that 1 hour yesterday and once there was no success, I used the alternative approach. 

widgetHandler does similar thing internally so it is not much difference